install-spark2/README.md

```diff
@@ -13,6 +13,9 @@ Non-working Components: Jupyter, livy (interactive)
 2. Run script action: `install-spark2-v01.sh` on this cluster. 
 
     Go to Azure portal > open cluster blade > open Script Actions tile > click Submit new and follow instructions. The script action is provided in this repository.
+    
+    When you are on the "Submit script action" blade, you will see "Bash script URI" field. You need to make sure that the `install-spark2-v01.sh` is stored in an Azure Storage Blob, and make the link public.
```

Comment:
This can be simplified by using the raw version of the script. i.e. https://raw.githubusercontent.com/hdinsight/script-actions/master/install-spark2/install-spark2-v01.sh